src/com/vivian/java/leetcode/Q28_strStr.java

```diff
@@ -8,7 +8,11 @@
  */
 public class Q28_strStr {
     public int strStr(String haystack, String needle) {
-        // TODO implement this
-        return 0;
+        for(int i=0;i<haystack.length();i++){
+            if(haystack.charAt(i)==needle.charAt(0)){
```

Comment:
Really? The 0th char in needle is the only thing you need to check?!

What test data did you use? 

How about this one? strStr("it doesn't work!", "or")
What's the expected result, and what's your actual result?